.github/workflows/nightly-latest-packages-validation.yml

```
@@ -0,0 +1,36 @@
+# In order to be updated when nightlies fail, please subscribe for updates on PR:
+# ---> https://github.com/facebook/metro/pull/1314 <---
+# where comments will be published on fail.
+
+# This is a bit of a workaround tackling the lack of an organic way
+# to notify certain people when a Github workflow fails:
+# https://github.com/orgs/community/discussions/18039
+
+name: facebook/metro/nightly-latest-packages-validation
+on:
+  schedule:
+    # Everyday at at 5:00 UTC (22:00 USA West Coast, 06:00 London)
```

nit: "London" assumes BST. UTC is good
```suggestion
    # Daily at at 5:00 UTC
```